.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+return (new \PhpCsFixer\Config())
+    ->setFinder(
+        \PhpCsFixer\Finder::create()
+            ->in([
+                getcwd().'/src',
+                getcwd().'/tests',
+            ])
+            ->exclude([
+                'migrations/*',
+            ])
+    )
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PSR12' => true,
+        '@Symfony' => true,
+        'align_multiline_comment' => true,
+        'array_indentation' => true,
+        'blank_line_after_opening_tag' => false, // allows for <?php declare(strict_types=1);
+        'class_attributes_separation' => ['elements' =>['property' => 'only_if_meta', 'const' => 'only_if_meta']],
+        'combine_consecutive_issets' => true,
+        'combine_consecutive_unsets' => true,
+        'comment_to_phpdoc' => true,
+        'date_time_immutable' => true,
+        'declare_strict_types' => true,
+        'doctrine_annotation_array_assignment' => true,
+        'doctrine_annotation_braces' => true,
+        'doctrine_annotation_indentation' => true,
+        'doctrine_annotation_spaces' => true,
+        'escape_implicit_backslashes' => true,
+        'explicit_indirect_variable' => true,
+        'explicit_string_variable' => true,
+        'final_internal_class' => false,
+        'general_phpdoc_annotation_remove' => false,
+        'header_comment' => false,
+        'heredoc_to_nowdoc' => false,
+        'linebreak_after_opening_tag' => false, // allows for <?php declare(strict_types=1);
```

Comment:
we got true here as well in our existing configs

Reply:
@veewee @janvernieuwe what is the consensus true or false for `blank_line_after_opening_tag` and `linebreak_after_opening_tag`?

Reply:
Make it true